### PR TITLE
fix: Fixed the FBW Logo + Vercel Logo appearing in the map page

### PR DESCRIPTION
**Pull request by LunakisDev**
## Summary of changes
Fixed the FBW Logo + Vercel Logo appearing in the map page (https://flybywiresim.com/map/) when the user zoon - unzoom.

## Screenshots
2r15JNzTrq

### CODE REVIEW

**Comment by Benjozork** [comment]
This is not how we should fix this, it should just be fixed with CSS

**Comment by LunakisDev** [comment]
Ah alright, Thanks for letting me know

**Comment by ItzARz** [comment]
I also managed to fix this issue with these simples changes 😄 : 

https://github.com/flybywiresim/website/commit/4f0ac6499c60c6580f07528fc45e153f513b0add

**Comment by Gaudv** [comment]
> This is not how we should fix this, it should just be fixed with CSS

@Benjozork changes have been made, ans it's now handled with css,

Noted that I also would recommand waiting for @ItzARz solution to be pulled into this PR, his proposition is better fitted.

File: src/pages/map/index.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import Section from '../../components/Utils/Section';
 const LeafletMap = dynamic(() => import('../../components/Map/Map'), { ssr: false });
 
 const Map: NextPage = () => (
-    <Section theme="dark" className="w-ful relative h-96 py-0">
+    <Section theme="dark" className="relative w-full h-[100vh] py-0">
         <LeafletMap className="w-full h-[100vh]" isFullPageMap />
```
Comment (NathanInnes) [inline src/pages/map/index.tsx:10]:
```suggestion
    <Section theme="dark" className="w-screen h-screen py-0">
        <LeafletMap className="w-full h-full" isFullPageMap />
```
Recommended changes

**Review by NathanInnes — APPROVED** [review]

**Pull request merged**